### ISSUE TITLE
Problems with logging in background

### ISSUE THREAD

**Issue by algrid** [Instabug/Instabug-iOS#420]
### Steps to Reproduce the Problem
Try to use `IBGLog` in background (background fetch).

### Expected Behavior
Logging should work.

### Actual Behavior
I see error messages like this:

>[Instabug] - IBGCore: error in saving IBGLog to CoreData: Error Domain=Application in background Code=404 "(null)". Memory: {
    total = "236.359375";
    used = "192.96875";
}

### Instabug integration code

        Instabug.start(withToken: token, invocationEvents: [.none])
        Instabug.willSendReportHandler = { report in
            // some code...
            return report
        }
        Instabug.tintColor = UIColor.someColor

### SDK Version
10.5.1

### iOS Version
14.4

### Device Model
any

**Comment by Kmohamed** [comment]
Yes, it will not be saved into a disk, but once the app came back to the foreground it will be saved on disk. 
Does this make sense?

**Comment by algrid** [comment]
@Kmohamed If it's not saved to disk, does it mean that it's stored in RAM? The app can be terminated by the OS (when the background fetch finishes) and then those logs will be lost?

**Comment by Kmohamed** [comment]
Yes, it will be saved in the RAM. But when the app terminated we remove old logs. So, it will be removed anyways.

**Comment by algrid** [comment]
I wanted to see some background logs coming with user reports and make sure that they're saved. 
But I guess I'm asking too much since anyway only recent logs are included with those reports. :) 

Also those "IBGCore: error" logs are a bit confusing - it looks like something's wrong while it's expected behavior. Anyway, it's not a big deal, thanks for clarification!

**Comment by Kmohamed** [comment]
I think you're right, we will work on improving these error messages 👍
